### PR TITLE
Feat/core logic preparation

### PR DESCRIPTION
**Pull request by jasonkwm**
## General Changes

- Core logic clean up
- Swap to arbitrum from bsc testnet
- Added Leverage feature
- Update env

### CODE REVIEW

File: src/maneki/modules/markets/ClaimRewardTopPanel.tsx
Comment (neosizzle) [inline src/maneki/modules/markets/ClaimRewardTopPanel.tsx:1]: REWARD_TOKEN_ADDR should be in env var

File: src/components/lists/ListMobileItem.tsx
Comment (neosizzle) [inline src/components/lists/ListMobileItem.tsx:1]: Screenshot 2023-08-02 at 10 12 50
clipping of percentage values in desktop

File: src/modules/markets/MarketAssetsListItem.tsx
Comment (neosizzle) [inline src/modules/markets/MarketAssetsListItem.tsx:1]: Screenshot 2023-08-02 at 10 12 50

Clipping with the 2x to 12x values

File: src/uiConfig.ts
Comment (neosizzle) [inline src/uiConfig.ts:1]: Where is the replacement for the logo?

**Review by neosizzle — CHANGES_REQUESTED** [review]
Changes needed on styling of the glp menu item and some env var abstractions

File: src/uiConfig.ts
Comment (jasonkwm) [reply src/uiConfig.ts:1]: Using a different method to get logo.

File: src/modules/markets/MarketAssetsListItem.tsx
Comment (jasonkwm) [reply src/modules/markets/MarketAssetsListItem.tsx:1]: added min width to prevent clipping

File: src/maneki/modules/markets/ClaimRewardTopPanel.tsx
Comment (jasonkwm) [reply src/maneki/modules/markets/ClaimRewardTopPanel.tsx:1]: done

**Review by neosizzle — APPROVED** [review]

**Pull request merged**